spanner_orm/tests/migrations_test.py

```diff
@@ -38,6 +43,16 @@ def test_retrieve(self):
                      migrations[0].migration_id)
 
   def test_generate(self):
+    testdata_filename = os.path.join(os.path.dirname(__file__),
+                                     'migrations')
+    shutil.rmtree(self.TEST_MIGRATIONS_DIR)
+    shutil.copytree(testdata_filename, self.TEST_MIGRATIONS_DIR)
+    os.chmod(self.TEST_MIGRATIONS_DIR,
+             stat.S_IROTH | stat.S_IWOTH | stat.S_IXOTH
+             | stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR)
+    for f in os.listdir(self.TEST_MIGRATIONS_DIR):
+      os.chmod(os.path.join(self.TEST_MIGRATIONS_DIR, f),
```

Comment:
we need to skip doing this if it's a directory, otherwise __pycache__ gets generated in the migrations dir after the first run and the shutil.rmtree dies because the __pycache__ dir in the temp directory has its execute bit stripped

Reply:
done